pages/flight-search.tsx

```diff
@@ -1,12 +1,40 @@
-import { useRouter } from 'next/router';
+import Link from "next/link";
+import { useState } from "react";
+import InputAutocomplete from "../components/inputAutocomplete";
+import { FlightSelectData } from "../models/FlightSelectData";
+import styles from "../styles/flight-search.module.scss";
 
-export default function FlightSearch() {
-  const router = useRouter();
 
+
+export default function FlightSearch(props: { flightSelectData: FlightSelectData}) {
+
+  const [fromFlightSelect, setFromFlightSelect] = useState("");
+  const [toFlightSelect, setToFlightSelect] = useState("");
+
+  const submitFromFlightSearch = (value: string) => {
+    setFromFlightSelect(value)
+  }
+  const submitToFlightSearch = (value: string) => {
+    setToFlightSelect(value);
+  }
+  const submitFlightSearch = () => {
+    alert('Submitting from location: ' + fromFlightSelect + '. Submitting to location: ' + toFlightSelect);
+  }
   return (
     <>
-      <div>Search Flights</div>
-      <button onClick={() => router.push('/flight-select')}>Search</button>
+    <div className={styles.flexContainer}>
+      <div>
+        <InputAutocomplete suggestions={props.flightSelectData.data.states.map((state) => state.name)} placeholder="From" onClickFlightReference={submitFromFlightSearch}></InputAutocomplete>
+      </div>
+      <div>
+        <InputAutocomplete suggestions={props.flightSelectData.data.states.map((state) => state.name)} placeholder="To" onClickFlightReference={submitToFlightSearch}></InputAutocomplete>
```

Comment:
Here I decided to pass the function down to the child so when the child get's clicked it comes back up to the parent but I think an alternative that we could do is store in context since what they search is going to be used probably in tons of places. Another alternative is we could store in Jotai which could then be accessed from tons of spots so I think it depends on how often we would retrieve this information.